spec/index.bs

```diff
@@ -7456,7 +7477,11 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
 
                     <dl class=switch>
                         : {{GPUStorageTextureAccess/"write-only"}}
-                        :: The access mode `A` is `write`.
+                        :: The access mode `A` is either `write` or `read_write`.
+                        : {{GPUStorageTextureAccess/"read-only"}}
+                        :: The access mode `A` is `read`.
+                        : {{GPUStorageTextureAccess/"read-write"}}
+                        :: The access mode `A` is `read_write`.
```

Comment:
I might be misreading but this looks inverted. WGSL read_write shouldn't work with a write-only binding.

Can a read-write binding work with any WGSL access mode?

```suggestion
                        :: The access mode `A` is `write`.
                        : {{GPUStorageTextureAccess/"read-only"}}
                        :: The access mode `A` is `read`.
```

Or are there weird API reasons you can't do that.
```suggestion
                        :: The access mode `A` is `write`.
                        : {{GPUStorageTextureAccess/"read-only"}}
                        :: The access mode `A` is `read`.
                        : {{GPUStorageTextureAccess/"read-write"}}
                        :: The access mode `A` is `read_write` or `write`.
```

Reply:
Ah yes, the only valid combination should be:

 - layout: `write-only`, WGSL: `write`
 - layout: `read-only`, WGSL: `read`
 - layout: `read-write`, WGSL: `read_write`
 - layout: `read-write`, WGSL: `write` Note that this is the only extra one due to the UAV/SRV thing mentioned in the other comment.